src/sentry/integrations/vercel/integration.py

```diff
@@ -316,8 +316,7 @@ def build_integration(self, state):
             external_id = data["user_id"]
             installation_type = "user"
             user = client.get_user()
-            name = user["name"]
-
+            name = user.get("name") or user["username"]
```

Comment:
I'm assuming the username will always be present ?

Reply:
@MeredithAnya yea, it's gotta be there for a user